app/views/layouts/application.html.slim

```diff
@@ -26,6 +26,9 @@ html lang="#{I18n.locale}"
       <script src='https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js'></script>
     <![endif]-->
 
+    - if Rails.env.production? && Figaro.env.google_analytics_key.present?
```

Comment:
I thought about that earlier, and I think you're right. Our default `application.yml` only includes the key in the production section.